src/blockdev.rs

```diff
@@ -798,7 +798,7 @@ fn lsblk_all(rereadpt: bool) -> Result<Vec<HashMap<String, String>>> {
     if rereadpt {
         for dev in output.lines() {
             if let Ok(mut fd) = std::fs::File::open(dev) {
-                let _ = reread_partition_table(&mut fd);
+                let _ = reread_partition_table(&mut fd, false);
```

Comment:
Maybe add a comment here?  Something like: "Best-effort reread of disk that may have busy partitions; don't retry"